src/v1/imports/carts/bigcommerce.ts

```diff
@@ -1,37 +1,38 @@
-import { xhrResponseSource } from "../../../shared/sources/xhr-response-source";
 import { QueryStringContext } from "../../../shared/types";
-import bigcommerceDataSource from "../../../shared/environment-data-sources/bigcommerce"
+import bigcommerceDataSource from "../../../shared/environment-data-sources/bigcommerce";
```

Comment:
I checked this file and it exports a `productId` for the `transactionEvent.items` , we should remove the `productId` since it doesn't belong in the interface

```js
                  return {
                    orderId: transaction.orderId.toString(),
                    productId: productId.toString(), //<--- remove this from bigCommerceDataSource
                    sku: sku.toString(),
                    name: (name || "N/A").toString(),
                    category: "N/A",
                    unitPrice: parseFloat(listPrice || 0),
                    quantity: parseInt(quantity || 1),
                    currency: "USD",
                  } as TransactionCartItem;
```